frontend/main.go

```diff
@@ -29,6 +29,13 @@ func main() {
 
 	logger.Info(fmt.Sprintf("%s (%s) started", ProgramName, version))
 
+	// Fetch the region from the env variable
+	region := os.Getenv("REGION")
```

Comment:
I think this is fine for now. Eventually we will have more external configuration for the service when running in a hosted environment and we may want to reconsider pulling from the env in an ad hoc manner